doc/user/content/sql/system-catalog.md

```diff
@@ -161,6 +161,19 @@ Field       | Type       | Meaning
 `name`      | [`text`]   | The name of the object.
 `type`      | [`text`]   | The type of the object: either `table`, `source`, `view`, `sink`, or `index`.
 
+### `mz_procs`
+
+The `mz_procs` table contains a row for each process/function in the system.
+
+Field          | Type       | Meaning
+---------------|------------|--------
+`id`           | [`text`]   | Materialize's unique ID for the type.
+`oid`          | [`oid`]    | A [PostgreSQL-compatible OID][oid] for the index.
+`schema_id`    | [`bigint`] | The ID of the schema to which the function belongs.
+`name`         | [`text`]   | The name of the function.
+`arg_types_oid` | [`_oid`]    | An array of [PostgreSQL-compatible OIDs][oid] representing the function's arguments' types.
```

Comment:
I think probably `oid array` per desire to use SQL standard names?

Reply:
Actually, crap, per design of the rest of the catalog these ought to be `text` columns that store the `GlobalId` of the entry in `mz_types`.